### PR TITLE
Works on ArcGIS clients

### PR DESCRIPTION
**Pull request by rgwozdz**
This PR:
1) Changes plugin class name to `VectorTileServer`, which leads to a change in route names necessary to work with ArcGIS clients
2) Add additional metadata and style routes needs for ArcGIS clients.
3) New tests
4) Readme updates

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+### Changed
```
Comment (dmfenton) [inline CHANGELOG.md]: if this is a breaking change there should be a heading here to denote that

Comment (rgwozdz) [reply CHANGELOG.md]: Adding "breaking change" as a header in 65327df.

File: src/esri-metadata.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const _ = require('lodash')
+const template = require('./esri-template-metadata')
+
+module.exports = function (req, res) {
+  // Adjust layer specific properties
+  this.model.pull(req, (e, geojson) => {
+    const { metadata } = geojson
```
Comment (haoliangyu) [inline src/esri-metadata.js]: Does Koop guarantee a `metadata` even if the provider does not add one?

Comment (rgwozdz) [reply src/esri-metadata.js]: This is a good question.  No, Koop doesn't guarantee it.  I have updated the code to make it fault tolerant for the case where a provider doesn't add a `metadata` property.

**Pull request merged**